pkg/services/ngalert/migration/models/alertmanager.go

```diff
@@ -0,0 +1,83 @@
+package models
+
+import (
+	"github.com/prometheus/alertmanager/config"
+	"github.com/prometheus/alertmanager/pkg/labels"
+	"github.com/prometheus/common/model"
+
+	apiModels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+	ngmodels "github.com/grafana/grafana/pkg/services/ngalert/models"
+)
+
+// UseLegacyChannelsLabel is a private label added to a rule's labels to enable routing to the nested route created
+// during migration.
+const UseLegacyChannelsLabel = "__use_legacy_channels__"
```

Comment:
I would like all labels we use in alert rules to be in one place in `ngalert/models`. This will let us keep track of all system-owned labels in the system.